core/feconf.py

```diff
@@ -136,37 +136,37 @@ class RteTypeTextAngularDict(TypedDict):
 # to PascalCase and its values to UPPER_CASE. Because we want to be consistent
```

Comment:
I assume this comment can now be deleted?

Reply:
Yup I'll go ahead and delete that :o